vscode-lean4/package.json

```diff
@@ -64,6 +64,11 @@
 					"default": true,
 					"markdownDescription": "Enable eager replacement of abbreviations that uniquely identify a symbol."
 				},
+				"lean4.editor.occurrenceHighlightWithStringFallback": {
```

Comment:
How about `fallBackToStringOccurrenceHighlighting`? Otherwise it is not as clear from the name alone that setting this to `false` does not disable all occurrence highlighting. I would also leave out the `editor` prefix.

Reply:
OK, I made the change.